pkg/template/master_template.go

```diff
@@ -293,6 +293,7 @@ systemd:
       Description=k8s-kubelet
       StartLimitIntervalSec=0
       [Service]
+      User=root
```

Comment:
So if you don't specify it what is the user running the service?

Reply:
It is still the root. But without `HOME` env variable. Here it is explained https://github.com/kubernetes/kubernetes/issues/45487#issuecomment-312042754